test/NeighbouringFileNavigator.test.ts

```
@@ -34,11 +34,30 @@ const setup = (children: Array<TAbstractFile>) => {
+};
+
 const setupFiles = (names: Array<string>) => {
 	const children = names.map((c) => createNote(c));
 	return setup(children) as TFile[];
 };
 
+const setupFolder = (names: Array<string>, parent?: TFolder) => {
+	const folder = createDir(names.join("-"));
+	const files = names.map((name) => createNote(name));
+	folder.children = files;
+	files.forEach((file) => (file.parent = folder));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)